crates/re_sdk/src/session.rs

```diff
@@ -296,3 +304,21 @@ impl std::borrow::Borrow<dyn LogSink> for Session {
         self.sink.as_ref()
     }
 }
+
+/// Trait to expose [`RecordingId`] from a session
+pub trait HasRecordingId {
+    fn get_recording_id(&self) -> RecordingId;
+}
+
+impl HasRecordingId for Session {
+    fn get_recording_id(&self) -> RecordingId {
+        self.recording_info
+            .as_ref()
+            .map_or(RecordingId::default(), |i| i.recording_id)
```

Comment:
Hum… using the default (`ZERO`) recording id seems sub-optimial - how would we get here?

Reply:
The Session builder *should* finalize the Session and always assign a session-id. If we're hitting this path it generally means somebody messed up. We could always `.unwrap()` :smiling_imp:

Reply:
Ok, I made it so that this isn't possible. The disabled Session uses default, but by virtue of being disabled that recording-id should never go anywhere.